src/helpers/render-document-file.js

```diff
@@ -350,13 +548,22 @@ async function findXMLEquivalent(docxDocumentInstance, vNode, xmlFragment) {
     for (let index = 0; index < vNode.children.length; index++) {
       const childVNode = vNode.children[index];
       // eslint-disable-next-line no-use-before-define
-      await convertVTreeToXML(docxDocumentInstance, childVNode, xmlFragment);
+      await convertVTreeToXML(docxDocumentInstance, childVNode, xmlFragment, imageOptions);
     }
   }
 }
 
 // eslint-disable-next-line consistent-return
-export async function convertVTreeToXML(docxDocumentInstance, vTree, xmlFragment) {
+export async function convertVTreeToXML(
+  docxDocumentInstance,
+  vTree,
+  xmlFragment,
+  imageOptions = null
+) {
+  // Use default options if not provided
+  if (!imageOptions) {
```

Comment:
I see that in line 595 we already set the image options in case they are not provided, which are passed here. Is there any case where you observed these were not set?